frontend/src/__tests__/components/home.test.js

```diff
@@ -0,0 +1,129 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import Home from '../../../pages';
+import { useGetBooks } from '../../../api/books';
+import userEvent from '@testing-library/user-event';
+jest.mock('../../../api/books');
+jest.mock('../../../api/authors');
+jest.mock(
+  'next/image',
+  () =>
+    function Image({ src, alt, width, height }) {
+      // eslint-disable-next-line @next/next/no-img-element
+      return (
```

Comment:
I would be curious to see if you are familiar with the Next Image component, and why we would want to use it. I personally think this capstone is a great use case for the Next Image component.

Reply:
From this: https://refine.dev/blog/using-next-image/#:~:text=One%20powerful%20feature%20of%20next,config. it seems like the automatic sizing of the image for different viewports is probably its most beneficial feature.

Reply:
Yeah the sizing is beneficial, but really that's not all that it does.